src/LibraryManager.Vsix/UI/Controls/Library.xaml.cs

```diff
@@ -257,5 +266,10 @@ private void ThisControl_GotFocus(object sender, RoutedEventArgs e)
                 LibrarySearchBox.Focus();
             }
         }
+
+        private void FilterOutUnmatchedItems(ref IEnumerable<CompletionItem>items, string versionPrefix)
+        {
+            items = items.Where(x => x.DisplayText.Contains(versionPrefix));
```

Comment:
Is completion Item for version only? If so, should we filter by x.DisplayText.StartsWith?